documentation/src/main/wiki/RunningSuite.md

```diff
@@ -1,18 +1,84 @@
 ## Running Suite
 
-Currently, running an AET suite requires using *aet-maven-plugin* which is an AET client application.
+To schedule AET suite execution request the [[AET Web API endpoint `/suite`|TestExecutor#run-suite]]
+with HTTP POST request that contains suite XML file content in the `suite` param.
+This action will trigger AET suite processing and will return information on how to track suite progress.
 
-### Requirements
+AET provides 2 tools to schedule suite execution:
+- [[Client Scripts|ClientScripts]]
+- [[Client Maven Plugin|ClientApplication]]
+however, you may use your own tool or just a simple `curl` to schedule the suite.
+
+### Running suite with bash script
+
+#### Requirements
+* Download the AET [shell script](https://github.com/Cognifide/aet/blob/master/client/client-scripts/aet.sh)
+  (see required prerequisites in [README](https://github.com/Cognifide/aet/blob/master/client/client-scripts/README.md))
+* Well-formed and valid xml test suite file available
+(described with details in [[Defining Suite|DefiningSuite]] chapter)
+
+#### Running from command line
+Running the AET suite with aet bash script from the command line can be
+done by invoking following command in the directory where the `aet.sh`
+file has been saved:
+
+```
+./aet.sh http://localhost:8181 my-suite.xml
+```
+
+The first param is the [[Test Executor|TestExecutor]] endpoint address and the second is
+the suite XML file. You can read more about the client script [[here|ClientScripts#usage]].
+
+During test suite processing there will be information on its progress displayed
+in the console. It reflects how many artifacts were currently collected, compared and reported.
+When processing is finished the information about the processing status:
+
+> Suite processing finished
+
+Read more about tracking progress in the [[Tracking Progress|TrackingProgress]] section.
+
+### Running suite with *aet-maven-plugin*
+
+**Important note**
+**`aet-maven-plugin` is deprecated and will be no longer supported. Expect it will be removed during next minor release.
```

Comment:
Removed during next minor release? It sounds scary :) Maybe we just use the word *soon* ?